deploy/ci.Jenkinsfile

```diff
@@ -35,15 +35,15 @@ pipeline {
                 cleanWs()
                 checkout scm
                 script {
-                    print('Set interim merge tag')
+                    print('Set interim merge branch')
                     sh """
-                    echo ${env.param_push_git_tag}
-                    if [ `git tag |grep -x ${env.mergeTag}` ]; then
+                    echo ${env.mergeTag}
```

Comment:
shouldn't we rename param as well? 
e.g env.mergeBranch

Reply:
Good point! Fixed